sql/schema.sql

```diff
@@ -0,0 +1,19 @@
+/* Create music_album table start */
+CREATE TABLE music_album ( 
+        id INTEGER GENERATED ALWAYS AS IDENTITY, 
+        published_date BOOLEAN, 
+        archived BOOLEAN, 
+        on_spotify BOOLEAN, 
+        genre_id INTEGER,
+        CONSTRAINT fk_genre FOREIGN KEY (genre_id) REFERENCES genre(id) ON DELETE CASCADE
+    PRIMARY KEY(id) 
+);
+/* Create music_album table end */
+
+/* Create genre table start */
+CREATE TABLE genre (
```

Comment:
I suggest that the `genre` table comes first before `music_album` because the latter is referencing the former which at that point would not be existing so an (`ERROR:  relation "genre" does not exist` ) error is raised.

Reply:
Thank you for pointing that out @AdornChoga, I have made requested changes